grid.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html>
+    <head>
+    <title>Noughts and Crosses</title>
+    <style>
+        .row div {
+            padding: 10px;
+            border: 1px solid black;
+            height:120px;
+            width: 120px;
+            float: left;
+        }
+        .row {
+            clear: both;
+        }
+    </style>
```

Comment:
There's more than one way to skin a cat and lots of different ways to achieve this. 
A few things to pick up on here:
1. This is fine for a demo but stylesheets should be in a separate file.
1. The .row div {} syntax affects all divs that are a children of the row element. That's fine and avoids having to put classes on all the sub divs. In this case though I would probably advise being more explicit. You might be adding extra divs you want to behave differently. Create a new class like .box or whatever and add it to the child divs. Note that this is just my opinion so feel free to ignore.
1. The clear both isn't really helping you.
1. To get it to appear centrally add a class to the outer div and give it the style of margin : 0 auto;

Here's what I had for this. Feel free to use it to compare. Don't feel you have to use this though.

`       
         .grid {
            margin: 0 auto;
            border: 1px solid black;
            width: 306px;
            height: 306px;
        }`
               `.box {
            border: 1px solid black;
            height: 100px;
            width: 100px;
            float: left;
        }`

Reply:
I AM AWESOME